test/inductor/test_flex_attention.py

```diff
@@ -525,6 +525,30 @@ def run_automatic_dynamic_test(
     def test_builtin_score_mods(self, dtype: torch.dtype, score_mod: Callable):
         self.run_test(score_mod, dtype)
 
+    @supported_platform
+    @patch.object(torch._inductor.config, "force_use_flex_attention", True)
+    @common_utils.parametrize("dtype", test_dtypes_fast)
+    @common_utils.parametrize("score_mod", test_score_mods)
+    def test_builtin_score_mods_seqlen_lt_default_sparse_block_size(
+        self, dtype: torch.dtype, score_mod: Callable
+    ):
+        # _DEFAULT_SPARSE_BLOCK_SIZE is 128
+        self.run_test(score_mod, dtype, B, H, 64, D, B, H, 64, D)
```

Comment:
Should you just some values that are like ... 63 instead? Seems like a harder test :)

Reply:
63 < BLOCK_N, which hit the bug I mentioned above. I'll add other tests for seqlen < BLOCK_M/N cases when I fix it.